build.gradle

```diff
@@ -337,10 +337,9 @@ dependencyManagement {
 dependencies {
 
     implementation group: 'com.github.hmcts', name: 'et-common', version: '2.4.2'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '3.17.6'
-
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '429eee3ceb'
```

Comment:
Can we get a released version in here please

Reply:
will do